Source/CommandLine.cpp

```diff
@@ -464,8 +481,11 @@ std::pair<juce::String, PluginTests::Options> parseCommandLine (const juce::Argu
 {
     auto fileOrID = args.getValueForOption ("--validate");
```

Comment:
This is the bit that actually needs to be fixed. Instead of using the `juce::ArgumentList::getValueForOption`, it should use my `getOptionValue`, in the same way as elsewhere in the file: `auto fileOrID = getOptionValue (args, "--validate", "", "Expected a plugin path for the --validate option").toString();`